client/src/components/APIDataTypes.ts

```diff
@@ -120,6 +120,15 @@ export type BuildingInfoRecord = {
   streetname: string;
   bldgclass: string;
   formatted_address: string;
+  /** The name of the NYCHA development (e.g. "SOTOMAYOR HOUSES").
+   * NULL if building is not part of NYCHA */
+  nycha_development: string | null;
+  /** Total executed residential evictions (since 2017) accross the building's entire NYCHA development.
+   * NULL values either mean 0 evictions took place across the development or building is not part of NYCHA. */
+  nycha_dev_evictions: number | null;
+  /** Total residential units accross the building's entire NYCHA development.
+   * NULL values either mean PLUTO listed 0 residential units across the development or building is not part of NYCHA. */
+  nycha_dev_unitsres: number | null;
```

Comment:
@toolness I updated these comments to be more descriptive of what NULL values mean in each field here

Reply:
Ohh nice that's great!  Cool, you can disregard my other comment about having a `nycha` object then, thanks!

Reply:
oh ok!